manifests/instance.pp

```diff
@@ -209,14 +207,12 @@
   String[1] $output_buffer_limit_slave                           = $redis::output_buffer_limit_slave,
   String[1] $output_buffer_limit_pubsub                          = $redis::output_buffer_limit_pubsub,
   String[1] $conf_template                                       = $redis::conf_template,
-  Stdlib::Absolutepath $config_dir                               = $redis::config_dir,
-  Stdlib::Filemode $config_dir_mode                              = $redis::config_dir_mode,
   Stdlib::Absolutepath $config_file                              = $redis::config_file,
   Stdlib::Filemode $config_file_mode                             = $redis::config_file_mode,
   Stdlib::Absolutepath $config_file_orig                         = $redis::config_file_orig,
   String[1] $config_group                                        = $redis::config_group,
   String[1] $config_owner                                        = $redis::config_owner,
-  Boolean $daemonize                                             = $redis::daemonize,
+  Boolean $daemonize                                             = true,
```

Comment:
The service scripts we have only work when daemonized. In config.pp we override it for the default to allow the unmanaged service on RH-based systems to work undaemonized.